Docker/ods-api-swaggerui/alpine/Dockerfile

```diff
@@ -33,10 +33,13 @@ RUN wget -q -O /app/SwaggerUI.zip https://pkgs.dev.azure.com/ed-fi-alliance/Ed-F
     rm -f /app/SwaggerUI.zip && \
     dos2unix /app/*.json && \
     dos2unix /app/*.sh && \
-    chmod 500 /app/run.sh && \
+    chmod 500 /app/run.sh -- ** && \
```

Comment:
-- ** is setting 500 to all files and folders in /app/ we only want to have 500 (read and execute) for run.sh since it is used to initialize the container